stable_baselines/common/vec_env/subproc_vec_env.py

```diff
@@ -161,3 +164,27 @@ def set_attr(self, attr_name, value, indices=None):
         for remote in [self.remotes[i] for i in indices]:
             remote.send(('set_attr', (attr_name, value)))
         return [remote.recv() for remote in [self.remotes[i] for i in indices]]
+
+
+def _flatten_obs(obs, space):
+    """
+    Flatten observations, depending on the observation space.
+
+    :param obs: (dict<ndarray>, tuple<ndarray> or ndarray) an observation.
+                Either a numpy array or a dict or tuple of numpy arrays.
+    :return (OrderedDict<ndarray>, tuple<ndarray> or ndarray) a flattened numpy array
+            or an OrderedDict or tuple of flattened numpy arrays.
+    """
+    assert isinstance(obs, list) or isinstance(obs, tuple)
```

Comment:
This does not match the docstring, no?

Also, you can simplify the check:

```python
assert isinstance(obs, (list, tuple))
```

Reply:
good catch, updated docstring.